agent/backend/pktvisor/pktvisor.go

```diff
@@ -258,6 +269,29 @@ func (p *pktvisorBackend) Start() error {
 	p.scraper = gocron.NewScheduler(time.UTC)
 	p.scraper.StartAsync()
 
+	//TODO create a startup for otel receiver / exporter component
```

Comment:
we discussed this needing to be featured flagged with a new configuration item in orb config, with a default to off. that way we can make these 0.10.0 release without using it by default and breaking any existing functionality